web3/contract.py

```diff
@@ -638,6 +638,14 @@ def _encode_constructor_data(cls, args=None, kwargs=None):
         return deploy_data
 
 
+def mk_collision_prop(fn_name):
+    def collision_fn():
+        msg = "Namespace collision for function name {0} with ConciseContract API.".format(fn_name)
+        raise AttributeError(msg)
+    collision_fn.__name__ = fn_name
+    return functools.partial(collision_fn)
```

Comment:
I'm not sure this `partial` is needed.

Reply:
you're right. it's a lazy left-over from when the collision_fn took signature still included  _self_ . thanks.